src/sql/parts/notebook/notebook.contribution.ts

```diff
@@ -44,6 +45,10 @@ export class NewNotebookAction extends Action {
 		let title = `Untitled-${counter++}`;
 		let untitledUri = URI.from({ scheme: Schemas.untitled, path: title });
 		let model = new NotebookInputModel(untitledUri, undefined, false, undefined);
+		if(model)
+		{
+			model.providerId = JUPYTER_NOTEBOOK_PROVIDER;
```

Comment:
This is very fixed - we can't handle extensibility with this? Could we instead try to handle it by file type instead since this should be "correct"? E.g.
- If no provider ID is defined, then find the provider for the 'IPYNB' filename, since that's our default file type. This lets us solve the "right" assumption which is that if we don't know what type of notebook it is, we should use a sensible file type & the currently registered provider.
- This will automatically pick up the Jupyter provider if present.